From 4399b28feb075c3d5a6f0619b24a96d42acc53f4 Mon Sep 17 00:00:00 2001
From: ndana <nathan.dana@radiantsolutions.com>
Date: Tue, 15 Sep 2020 15:44:28 -0600
Subject: [PATCH] change extension

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 449b74e..000b94f 100644
--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@ setup(
     url="https://github.com/martinResearch/DEODR",
     license="BSD",
     packages=find_packages(),
-    package_data={"deodr": ["*.pyx", "data/*.*", "data/**/*.*"]},
+    package_data={"deodr": ["*.pyx", "*pxd", "data/*.*", "data/**/*.*"]},
     ext_modules=my_modules,  # additional source file(s)),
     include_dirs=[np.get_include()],
     setup_requires=["numpy", "scipy", "cython"],
-- 
2.28.0

