From ef1382c415830aef4d9b89869fc739a1efa2e779 Mon Sep 17 00:00:00 2001
From: ndana <nathan.dana@radiantsolutions.com>
Date: Tue, 15 Sep 2020 09:15:08 -0600
Subject: [PATCH] use debug ext

---
 setup.py | 21 +++++++++++----------
 1 file changed, 11 insertions(+), 10 deletions(-)

diff --git a/setup.py b/setup.py
index 449b74e..f3497e5 100644
--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,7 @@
 """Setup script for the DEODR project."""
 
 from setuptools import setup, find_packages
+from distutils.core import Extension
 
 from Cython.Build import cythonize
 
@@ -8,16 +9,16 @@ import numpy as np
 
 
 # compilation mode for debuging
-# extensions = [
-# Extension("differentiable_renderer_cython",
-# ["DEODR/differentiable_renderer_cython.pyx"]
-# ,extra_compile_args=["-Zi", "/Od"]
-# ,extra_link_args=["-debug"],
-# undef_macros = [ "NDEBUG" ]
-# )
-# ]
-
-extensions = "deodr/differentiable_renderer_cython.pyx"
+extensions = [
+    Extension("differentiable_renderer_cython",
+        ["deodr/differentiable_renderer_cython.pyx"],
+        extra_compile_args=["-Zi", "/Od"],
+        extra_link_args=["-debug"],
+        undef_macros = [ "NDEBUG" ]
+    )
+]
+
+# extensions = "deodr/differentiable_renderer_cython.pyx"
 
 my_modules = cythonize(extensions, annotate=True, language="c++")
 
-- 
2.28.0

